pkg/skaffold/build/jib/sync.go

```diff
@@ -46,6 +53,131 @@ type JSONSyncEntry struct {
 	Dest string `json:"dest"`
 }
 
+func InitSync(ctx context.Context, workspace string, a *latest.JibArtifact) error {
+	syncMap, err := getSyncMapFunc(ctx, workspace, a)
+	if err != nil {
+		return err
+	}
+	syncLists[getProjectKey(workspace, a)] = *syncMap
+	return nil
+}
+
+// returns toCopy, toDelete, error
+func GetSyncDiff(ctx context.Context, workspace string, a *latest.JibArtifact, e filemon.Events) (map[string][]string, map[string][]string, error) {
+	// if anything that was modified was a buildfile, do NOT sync, do a rebuild
+	buildFiles := GetBuildDefinitions(workspace, a)
+	for _, f := range e.Modified {
+		f, err := toAbs(f)
+		if err != nil {
+			return nil, nil, errors.WithStack(err)
```

Comment:
We have not used `errors.WithStack` elsewhere in the code. Can we use `errors.Wrap` like elsewhere here?